composer.json

```diff
@@ -27,7 +27,7 @@
         "illuminate/contracts": "^6|^7|^8",
         "illuminate/http": "^6|^7|^8",
         "illuminate/support": "^6|^7|^8",
-        "lcobucci/jwt": "^3.2",
+        "lcobucci/jwt": "^3.2 <3.4",
```

Comment:
this does not permit the upgrade to PHP 8 since lcobucci/jwt permits it at version 4.0 and above:
https://github.com/lcobucci/jwt/blob/4.1.4/composer.json